pkgs/test_core/lib/src/runner/spawn_hybrid.dart

```diff
@@ -19,7 +21,8 @@ import 'package:test_api/src/util/remote_exception.dart'; // ignore: implementat
 /// This connects the main isolate to the hybrid isolate, whereas
 /// `lib/src/frontend/spawn_hybrid.dart` connects the test isolate to the main
 /// isolate.
-StreamChannel spawnHybridUri(String url, Object? message) {
+StreamChannel spawnHybridUri(String url, Object? message, Suite suite) {
```

Comment:
Is this a breaking change?

Reply:
This is an internal only method (it is what actually spawns the isolate). Confusing because it has the same name as the public one though.